product_state/views/product_template.xml

```diff
@@ -144,7 +144,7 @@
         <field name="priority">50</field>
         <field name="arch" type="xml">
             <field name="type" position="after">
-                <field name="state" />
+                <field name="product_state_id" />
```

Comment:
Why this one instead of the other?

Reply:
state is just a compute of product_state_id.code <br/>
You can use more odoo function by showing directly the many2one